.travis.yml

```diff
@@ -74,7 +74,8 @@ install:
 - source scripts/setup.sh || exit 1
 - source scripts/setup_gae.sh || exit 1
 
-script:
+bundler_args: --retry 1
```

Comment:
but will the retry still apply to the tests then?

On Fri, Jul 29, 2016 at 2:10 AM, Barnabas Makonda <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/oppia/oppia/pull/2237#discussion_r72745553>:
>
> > @@ -74,7 +74,8 @@ install:
> >  - source scripts/setup.sh || exit 1
> >  - source scripts/setup_gae.sh || exit 1
> >
> > -script:
> > +bundler_args: --retry 1
>
> only bundler_args
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/2237/files/c87076de023f037b5e53f7222cd19c00d83d7310#r72745553>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGIsJH6586on-VXmIOn9yReZHDF_v-VIks5qaZlXgaJpZM4JHrA1>
> .
>